pyvista/core/grid.py

```diff
@@ -764,3 +764,41 @@ def gen_coords(i):
         grid.field_data.update(self.field_data)
         grid.copy_meta_from(self, deep=True)
         return grid
+
+    @property
+    def extent(self) -> tuple:
+        """Return or set the extent of the UniformGrid.
+
+        This is the same as :class:`Grid.dimensions` with the upper bounds
+        minus one.
```

Comment:
To be honest this sentence is very confusing to me. I _think_ what it means is that if along a given axis we have an extent of `(i, j)` then the corresponding dimension is `j - i + 1`, but I only gathered this from the examples. But to be honest the whole concept of `extent` seems a bit confusing to me.